test/test-e2e-docker.sh

```diff
@@ -88,7 +88,7 @@ curl -s $APP/this/is/a/test \
   && echo 'ok # echo server responded' \
   || echo 'not ok # echo server failed to respond'
 
-../../bin/sl-pmctl.js -C $STRONGLOOP_PM env-set foo=success bar=foo \
+../../node_modules/strong-mesh-models/bin/sl-meshctl.js -C $STRONGLOOP_PM env-set foo=success bar=foo \
```

Comment:
The tests should probably still use the wrapper recommended for dealing with fragility.. otherwise what are you testing? ;-)